package.json

```diff
@@ -16,6 +16,7 @@
     "storybook": "storybook dev -p 6006",
     "build-storybook": "storybook build",
     "generate:api": "openapi-codegen gen api",
+    "generate:jobService": "openapi-codegen gen jobService",
     "generate:userService": "openapi-codegen gen userService",
     "generate:services": "npm run generate:userService",
```

Comment:
please add it to this one too. Also, I should have used `yarn` here, not `npm run`, so please change that as well:
```
"generate:services": "yarn generate:userService && yarn generate:jobService"
```